airflow-core/src/airflow/ui/src/i18n/README.md

```diff
@@ -17,8 +17,188 @@
+
+## 4. Requirements
+
+### 4.1. Translation ownership and code ownership
+
+Each supported translation, except for the default language, MUST meet one of the following conditions at any given time (except for transition times in section 6.4.):
+
+- Have at least one translation owner who is also a code owner.
+- Have at least one translation owner, with a translation sponsor assigned as a code owner.
+
+### 4.2. Approval of translation owners candidates
+
+- Translation owners candidates MUST declare and demonstrate a sufficient level of proficiency in the target language for translation purposes, including technical terminology.
+- Translation owners candidates, who are non-committers, MUST also meet the following criteria:
+     - They are active long-term contributors to the Apache Airflow project at the time of request.
+     - They have basic skills of working with GIT and GitHub, as well as modifying JSON translation files within their target language.
+     - They have the support of a Apache Airflow committer who will act as a translation sponsor.
+- Translation owners candidates MUST go through the approval process detailed in section 6.1.
+
+### 4.3. Approval of new translations
+
+To accept a new translation to the codebase, it MUST be approved through a review process discussed in section 6.2.
+
+### 4.4. Resolution of translation conflicts
+
+Translation conflicts MUST be resolved according to the procedures outlined in section 6.3.
+
+### 4.5. Adding / rephrasing terms
+
+- When new terms are added to the default language, all translation owners SHOULD create a follow-up PR to comply with the changes in their assigned language within a reasonable time.
+- When rephrasing terms in the default language, all translation owners SHOULD do the same as above, **if needed**.
+
+> [!NOTE]
+> Tooling for detecting missing translations is available (see Tools & Resources section below).
+
+### 4.6. Deprecating / refactoring terms
+
+When existing terms are deprecated or refactored (key renamed/relocated but value unchanged), **the contributor initiating the change is responsible for updating all language files, and not the translation/code owner**. Automation through Breeze tooling should be used when available.
+
+### 4.7. Approval of translation-related Pull Requests (PRs)
+
+- If the code owner is also a translation owner of the respective translation:
+    - Others' PRs should be approved and merged normally by them.
+    - Their own PRs should be approved and merged normally by another committer, preferably another code owner in their locale (if such exists).
+- Otherwise, if the code owner is sponsored:
+    - They should merge the translation-related PRs only after they are approved by the translation owner.
+- Other committers may review and approve translation-related PRs in any aspects, but they SHOULD NOT merge them without the approval of the translation owner and the consent of the code owner.
+
+## 5. Roles & responsibilities
```

Comment:
Should we move this section above? I was confused when reading 4 without these context